platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -239,6 +240,28 @@ whose names differ from the style specification are listed below:
 <% for (const type in renamedProperties) { -%>
 <% if (renamedProperties.hasOwnProperty(type)) { -%>
 
+### <%- camelize(type) %> style functions
+
+The runtime styling API introduced `MGLStyleFunction` to the <%- os %> SDK.
+[Data-driven styling](data-driven-styling.html) expands `MGLStyleFunction` and
+introduces `MGLInterpolationMode`. Individual style property documentation
+includes which subclasses of `MGLStyleFunction` are enabled for that property.
+
+In style specification       | In the SDK
+-----------------------------|-----------
+`zoom function`              | `MGLCameraStyleFunction`
+`property function`          | `MGLSourceStyleFunction`
+`zoom-and-property functions`| `MGLCompositeStyleFunction`
+
+### <%- camelize(type) %> interpolation mode
```

Comment:
This table doesn't need its own subheading, but a complete sentence introducing the concept of an interpolation mode is would help to make this document flow better.